### ISSUE TITLE
Incompatibility with Fortinet Device Guard

### ISSUE THREAD

**Issue by dorssel** [dorssel/usbipd-win#322]
@dorssel Is it possible that our antivirus software is causing problems with unloading the drivers during --force? We use forticlient. Our software seems to work ok on machines that don't have it installed. But I have not had a wide enough test yet to prove that out. Just wondering what you think? 

All devices that are on the USB have the forticlient driver added to their list of drivers.

![image](https://user-images.githubusercontent.com/5494992/159470487-d568b08d-5ece-478c-9bef-686381d40ade.png)

_Originally posted by @d0n13 in https://github.com/dorssel/usbipd-win/issues/270#issuecomment-1075047979_

**Comment by d0n13** [comment]
I finally got forticlient removed. Issue remains...

**Comment by dorssel** [comment]
To confirm, does the property page indeed list only `winusb.sys`?

**Comment by d0n13** [comment]
![image](https://user-images.githubusercontent.com/5494992/160821127-2d62e8ba-666f-44a9-bf00-9d31ee2391af.png)

Ok, this is probably unrelated to the issue exactly but just noticed it when getting the screenshot.

The device is listed as a portable device now. It's a pixel phone. It used to appear only under USB serial devices. Is that a driver side effect? 
Also noted that it's saying the device requires further installation. I'm not sure if I know what that means.

<img width="439" alt="image" src="https://user-images.githubusercontent.com/5494992/160818506-47a8bdd9-2c9b-4c03-901c-2f1a89d59ed3.png">

The message logged as part of driver loading under "view all events" is as follows. 

```
Device USB\VID_18D1&PID_4EE6&MI_00\7&2d18d080&b&0000 requires further installation.
Driver Management has concluded the process to add Service WUDFWpdMtp for Device Instance ID USB\VID_18D1&PID_4EE6&MI_00\7&2D18D080&B&0000 with the following status: 0.
Driver Management has concluded the process to add Service WinUsb for Device Instance ID USB\VID_18D1&PID_4EE6&MI_00\7&2D18D080&B&0000 with the following status: 0.
```

**Comment by dorssel** [comment]
That makes sense. The normal PnP driver installation for the Windows host driver didn't complete yet. That's why it is probably not possible to uninstall and replace the driver with `--force`, because the device is marked for "installation in progress" already. A reboot may help, or else: try disabling the device and reboot and then --force. Or maybe try to uninstall the current drivers manually.

**Comment by dorssel** [comment]
Now that we resolved #325 with PR #327, maybe Fortinet wasn't the problem after all? Can we somehow confirm that this issue is actually real?

**Comment by d0n13** [comment]
Forticlient is not the issue I don't think either. As I noticed when I removed it.
